tools/accuracy_checker/configs/mosaic.yml

```diff
@@ -0,0 +1,35 @@
+models:
+  - name: style_transfer
+
+    launchers:
+
+      - framework: onnx_runtime
+        device: CPU
+        model: public/mosaic/mosaic.onnx
+        adapter: style_transfer
+
+      - framework: dlsdk
+        device: CPU
```

Comment:
please remove device (we prefer to provide it in commad line using -td option
and add tags for opportunity to filter model by precision